app/content/models/registration.py

```diff
@@ -51,30 +52,37 @@ class Meta:
 
     @classmethod
     def has_retrieve_permission(cls, request):
-        return check_has_access(cls.has_retrieve_access, request,)
+        return True
 
     @classmethod
     def has_list_permission(cls, request):
-        return check_has_access(cls.has_access, request,)
+        return cls.has_event_permission(cls, request)
 
-    @staticmethod
-    def has_write_permission(request):
+    @classmethod
+    def has_write_permission(cls, request):
         return bool(request.user)
 
-    @staticmethod
-    def has_create_permission(request):
-        return request.id is not None
+    def has_event_permission(self, request):
+        if request.id is None:
```

Comment:
Du sjal ikke trenge å sette user id for hvis den er noen så er det ikke noe bruker eller

Reply:
Så jeg skal bare kjøre set_user_id uten sjekken? Eller returnere False  hvis request.id er None?

Reply:
Mente egentlig at du ikke skal trenge å sette den